tasks/install-pcf-gcp/terraform/sql.tf

```diff
@@ -96,6 +96,30 @@ resource "google_sql_database" "diego" {
   count = "1"
 }
 
+resource "google_sql_database" "account" {
+  name       = "diego"
+  depends_on = ["google_sql_database.diego"]
+  instance   = "${google_sql_database_instance.master.name}"
+
+  count = "1"
+}
+
+resource "google_sql_database" "nfsvolume" {
+  name       = "diego"
+  depends_on = ["google_sql_database.account"]
+  instance   = "${google_sql_database_instance.master.name}"
+
+  count = "1"
+}
+
+resource "google_sql_database" "networkpolicyserver" {
+  name       = "diego"
+  depends_on = ["google_sql_database.nfsvolume"]
```

Comment:
No, but I see that it is being ordered by the previous database, I believe just an attempt to sequence it.